[BUG] Master branch is failing to build

#### Bug Report Checklist

The master branch is failing to build on my machine, is anyone else having this issue?
Git commit: 3739584a1a74d8cb124c38d3b03923161e01a4fe
Steps to repoduce:
- `git clone git@github.com:OpenAPITools/openapi-generator.git`
- `cd openapi-generator/`
- `mvn clean install`
Errors are printed in the terminal:
```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running TestSuite
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[ERROR] Tests run: 615, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 22.008 s <<< FAILURE! - in TestSuite
[ERROR] testGenerateOperationBodyWithCodedTestData(org.openapitools.codegen.java.jaxrs.JavaJAXRSCXFExtServerCodegenTest)  Time elapsed: 0.358 s  <<< FAILURE!
java.lang.AssertionError: expected object to not be null
	at org.openapitools.codegen.java.jaxrs.JavaJAXRSCXFExtServerCodegenTest.checkFile(JavaJAXRSCXFExtServerCodegenTest.java:393)
	at org.openapitools.codegen.java.jaxrs.JavaJAXRSCXFExtServerCodegenTest.testGenerateOperationBodyWithCodedTestData(JavaJAXRSCXFExtServerCodegenTest.java:664)

[ERROR] testGenerateOperationBodyWithJsonTestData(org.openapitools.codegen.java.jaxrs.JavaJAXRSCXFExtServerCodegenTest)  Time elapsed: 0.377 s  <<< FAILURE!
java.lang.AssertionError: expected object to not be null
	at org.openapitools.codegen.java.jaxrs.JavaJAXRSCXFExtServerCodegenTest.checkFile(JavaJAXRSCXFExtServerCodegenTest.java:393)
	at org.openapitools.codegen.java.jaxrs.JavaJAXRSCXFExtServerCodegenTest.testGenerateOperationBodyWithJsonTestData(JavaJAXRSCXFExtServerCodegenTest.java:711)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   JavaJAXRSCXFExtServerCodegenTest.testGenerateOperationBodyWithCodedTestData:664->checkFile:393 expected object to not be null
[ERROR]   JavaJAXRSCXFExtServerCodegenTest.testGenerateOperationBodyWithJsonTestData:711->checkFile:393 expected object to not be null
[INFO] 
[ERROR] Tests run: 615, Failures: 2, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for openapi-generator-project 4.0.0-SNAPSHOT:
[INFO] 
[INFO] openapi-generator-project .......................... SUCCESS [  3.126 s]
[INFO] openapi-generator (core library) ................... FAILURE [ 51.314 s]
[INFO] openapi-generator (executable) ..................... SKIPPED
[INFO] openapi-generator (maven-plugin) ................... SKIPPED
[INFO] openapi-generator-gradle-plugin (maven wrapper) .... SKIPPED
[INFO] openapi-generator-online ........................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  55.289 s
[INFO] Finished at: 2019-03-08T22:56:09-08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.0:test (test-testng) on project openapi-generator: There are test failures.
[ERROR] 
[ERROR] Please refer to /Users/justin/programming/open_source/openapi-generator/modules/openapi-generator/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
```

Environment:
java version "1.8.0_121"
Java(TM) SE Runtime Environment (build 1.8.0_121-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.121-b13, mixed mode)
Apache Maven 3.6.0 (97c98ec64a1fdfee7767ce5ffb20918da4f719f3; 2018-10-24T11:41:47-07:00)
Maven home: /usr/local/Cellar/maven/3.6.0/libexec
Java version: 1.8.0_121, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk1.8.0_121.jdk/Contents/Home/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.13.6", arch: "x86_64", family: "mac"

--- a/modules/openapi-generator/src/test/java/org/openapitools/codegen/java/jaxrs/JavaJAXRSCXFExtServerCodegenTest.java
+++ b/modules/openapi-generator/src/test/java/org/openapitools/codegen/java/jaxrs/JavaJAXRSCXFExtServerCodegenTest.java
@@ -558,7 +558,7 @@ public class JavaJAXRSCXFExtServerCodegenTest {
 
     @Test
     public void testAddOperationToGroup() throws Exception {
-        File output = Files.createTempDirectory("test").toFile();
+        File output = Files.createTempDirectory("test").toFile().getCanonicalFile();
         output.deleteOnExit();
 
         OpenAPI openAPI = new OpenAPIParser().readLocation("src/test/resources/3_0/tags.yaml", null, new ParseOptions())
@@ -634,11 +634,11 @@ public class JavaJAXRSCXFExtServerCodegenTest {
         assertEquals(group5List.get(1).subresourceOperation, true);
     }
 
-    @Test(enabled = false)
+    @Test()
     public void testGenerateOperationBodyWithCodedTestData() throws Exception {
-        File output = Files.createTempDirectory("test").toFile();
+        File output = Files.createTempDirectory("test").toFile().getCanonicalFile();
         output.deleteOnExit();
-        String outputPath = output.getCanonicalPath().replace('\\', '/');
+        String outputPath = output.getAbsolutePath().replace('\\', '/');
 
         OpenAPI openAPI = new OpenAPIParser()
                 .readLocation("src/test/resources/3_0/petstore.yaml", null, new ParseOptions()).getOpenAPI();
@@ -680,11 +680,11 @@ public class JavaJAXRSCXFExtServerCodegenTest {
         checkFile(generator, outputPath + "/test-data-control.json", false);
     }
 
-    @Test(enabled = false)
+    @Test()
     public void testGenerateOperationBodyWithJsonTestData() throws Exception {
-        File output = Files.createTempDirectory("test").toFile();
+        File output = Files.createTempDirectory("test").toFile().getCanonicalFile();
         output.deleteOnExit();
-        String outputPath = output.getCanonicalPath().replace('\\', '/');
+        String outputPath = output.getAbsolutePath().replace('\\', '/');
 
         OpenAPI openAPI = new OpenAPIParser()
                 .readLocation("src/test/resources/3_0/petstore.yaml", null, new ParseOptions()).getOpenAPI();

